packages/core/test/runScenarios.spec.mjs

```diff
@@ -0,0 +1,26 @@
+// @ts-check
+
+import test from 'ava'
```

Comment:
Why was the switch to esm necessary? 

If only for top-level await, then we can wrap it in an async function and stay within cjs

Reply:
Would also prefer if this can be done without migrating the test suite to ESM. Is that a bug in ava...?

Reply:
I'll try wrapping it in an async function and staying in CJS, but Ava may not support it.